samples/contractupdate/src/util.ts

```diff
@@ -3,5 +3,8 @@ export function setErrorString(setError: (err: string) => void) {
 }
 
 export function errorString(err: any): string {
-    return (err as Error).message || (err as string);
+    if (err instanceof Error) {
+        return err.message;
+    }
+    return err.toString();
```

Comment:
This fix surfaces an underlying issue and must therefore be done in a separate PR in a way that addresses the following:

On errors like "user rejected the request", WalletConnect throws the reject object (which is not an `Error`). This has worked until now because this object has a field `message`, so the function would "luckily" interpret it as `Error` and extract this message. This changes with this fix and the object will now just render as `[object Object]`. Passing `err as string` like before will fail because we're then trying to render an object which is not allowed in React.

Reply:
Task: https://concordium.atlassian.net/browse/CBW-1148

Reply:
Error-related changes moved to #44.